src/fiat-connect-client.ts

```diff
@@ -140,6 +145,20 @@ export class FiatConnectClientImpl implements FiatConnectApiClient {
         return handleError(data)
       }
 
+      const headerSetCookie = response.headers.get('set-cookie')
+
+      if (headerSetCookie) {
+        headerSetCookie.split(',').forEach(async (cookie) => {
+          await this.cookieJar.setCookie(
```

Comment:
What's the behavior of the resulting cookie jar if we end up setting the same cookie twice? For example, if we login twice, will the second time we set a previously-set cookieoverwrite the previous value?

Reply:
If the cookie keys are the same, the previous cookie's value is overridden.